src/Monitor/Monitor.Test/Monitor.Test.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Monitor" Version="0.21.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Monitor" Version="0.22.0-preview" />
```

Comment:
Should I raise a separate PR for this or Make the changes in Sql.csproj file in this PR?

Reply:
@AnanyShah the PR with this change was just merged -- I would suggest pulling the latest from the `master` branch to get these changes and make sure that it runs successfully on CI

Reply:
@cormacpayne The checks are failing because the src/Sql/Sql.Test/Sql.Test.csproj still has dependency on Monitor SDK. Will it be removed, or do I need to update the monitor SDK version in the Sql.Test.csproj?
If I need to update the monitor SDK version in the Sql.Test.csproj should I do that change in this PR or raise a seperate PR for that.

Reply:
@AnanyShah the dependency on the Monitor SDK won't be removed from the Sql test project (since they use the actual module in their tests), so the version there will need to be bumped. I'll push a commit in your branch that does this update.